### PR TITLE
Bugfix - collect LoqusDB instances correctly in institute settings

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR:
- Prevents a bug caused by the server settings form initialized before the loqus instance settings and resulting in an empty Loqus instances select in institute settings
- Removes some code from #2349, that looks intimidating already.

**Prepare for test, locally**:
- Edit demo config file by uncomment the following lines:
```
LOQUSDB_SETTINGS = [
     {"binary_path": "/miniconda3/envs/loqus2.5/bin/loqusdb", "id": "default"},
     {"binary_path": "/miniconda3/envs/loqus2.5/bin/loqusdb", "id": "hejhej"},
     {"binary_path": "/bin/yetanother/loqusdb", "id": "Mayo"},
 ]
```
- Clone and install in a dedicated environment [loqusDB executable](https://github.com/moonso/loqusdb).
- Replace the binary_path of the 3 instances in the settings with the path to the loqus binary, something like this: `/Users/your-username/miniconda3/envs/loqus_env_name/bin/loqusdb`

**How to test**:
1. On master branch start scout demo and go to [institute settings page](http://localhost:5000/overview/cust000/settings)
2. Note that the the loqusdb id select has only one option: "default".
3. Switch to this branch and the 3 options should be shown
4. Test the branch on clinical-db stage and make sure that variants have observations

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Testing using master:

image


Testing using this branch: 🆗 

image

File: scout/server/blueprints/institutes/forms.py [hunk 1]
```diff
@@ -74,8 +81,7 @@ class InstituteForm(FlaskForm):
     )
     institutes = NonValidatingSelectMultipleField("Institutes to share cases with", choices=[])
 
-    loqus_instances = [instance["id"] for instance in loqusdb.loqusdb_settings]
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/forms.py:77]: This code gets called immediately when the app launches, before the code that collects the loqus instances in loqus extension, so it just gets a "default" value.

File: scout/server/blueprints/institutes/views.py [hunk 2]
```diff
@@ -246,12 +246,14 @@ def institute_settings(institute_id):
             return redirect(request.referrer)
 
     data = controllers.institute(store, institute_id)
+    loqus_instances = loqusdb.loqus_ids if hasattr(loqusdb, "loqus_ids") else []
```
Comment (northwestwitch) [inline scout/server/blueprints/institutes/views.py:249]: Populate loqus_instances list after the loqus extension is called and instances are saved

**Comment by northwestwitch** [comment]
I'd really like to squeeze this PR in the next scout release, because it's fixing a bug that we already have in master. Do you have time @dnil?

**Comment by dnil** [comment]
No, but I'm a little bored so will look anyway. 😄

File: scout/server/extensions/loqus_extension.py [hunk 3]
```diff
@@ -63,6 +64,7 @@ def init_app(self, app):
         """Initialize from Flask."""
         LOG.info("Init loqusdb app")
         self.loqusdb_settings = self.app_config(app)
+        self.loqus_ids = [setting.get("id", "default") for setting in self.loqusdb_settings]
```
Comment (dnil) [inline scout/server/extensions/loqus_extension.py:67]: 👍

**Review by dnil — APPROVED** [review]

**Pull request merged**